src/Cms/AbstractWebTemplate.php

```diff
@@ -289,4 +360,93 @@ public function twitterCardImage()
 
         return $this->metaImage();
     }
+
+
+
+    // Polylingual
```

Comment:
I propose we move these methods to a `LocaleAwareTrait` except for `AbstractWebTemplate::currentLanguage()` and `AbstractWebTemplate::currentLocale()`.